### PR TITLE
LPS-100140 Create and migrate AssetCategoriesSelector to react

### PR DESCRIPTION
**Pull request by carloslancha**

### CODE REVIEW

**Comment by julien** [comment]
Just started reviewing :)

File: modules/apps/asset/asset-taglib/package.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
 	"dependencies": {
+		"classnames": "2.2.6",
```
Comment (julien) [inline modules/apps/asset/asset-taglib/package.json]: There's a missing `main` property and the corresponding `index.es.js` here, I'll add it (locally) and we can review once the PR for LPS-100143 is sent.

**Comment by carloslancha** [comment]
ci:test:relevant

**Comment by carloslancha** [comment]
ci:test:sf

**Comment by carloslancha** [comment]
ci:test:sf

**Comment by carloslancha** [comment]
ci:test:sf

**Comment by carloslancha** [comment]
ci:test:sf

**Comment by carloslancha** [comment]
Resent: https://github.com/jbalsas/liferay-portal/pull/1974

**Pull request closed without merging**